src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx

```diff
@@ -129,13 +129,33 @@ const useDataTree = ({
     const style = item?.background ?? item?.getOneColor?.();
     const color = chroma(style ?? '#666').alpha(1);
     const mods: Record<string, any> = { hidden, type, isDrawing };
+
     const label = (() => {
       if (!type) {
         return 'No Label';
       } else if (type.includes('label')) {
         return item.value;
       } else if (type.includes('region') || type.includes('range')) {
-        return (item?.labels ?? []).join(', ') || 'No label';
+        const labelsInResults = item.results
+          .filter((result: any) => result.type.endsWith('labels'))
+          .map((result: any) => result.selectedLabels || []);
```

Comment:
the problem here is that `selectedLabels` can be not anly an array, but a simple label (when related labelset has `allowEmpty`), so it's better to check for it.

Reply:
Can you give me an exemple? I try to reproduce the selectedLabels returning a simple label to understand what I should check, but I wasn't able to. Every test that I made it's returning an array or returning undefined.